### ISSUE TITLE
Test hung with snRuntime-rtl-snRuntime-rtl-tls

### ISSUE THREAD

**Issue by kevinyuan** [pulp-platform/spatz#4]
Dear spatz team,

I followed the instructions and compiled the platform on Unbuntu 22.04 without seeing any obvious error.

However when I run "make sw.test.vlt", the test hung at the following position for 1 hour.

```
cd sw/build && make test
make[1]: Entering directory '/home/kevin/works/spatz/hw/system/spatz_cluster/sw/build'
Running tests...
Test project /home/kevin/works/spatz/hw/system/spatz_cluster/sw/build
      Start  1: snRuntime-rtl-snRuntime-rtl-tls

```

I very much appreciate it if you could provide tips to fix this issue.

Best regards .

Kevin

**Comment by xiaorui-yin** [comment]
I encountered the same issue, but I noticed there was an error when compiling the software
```
error: address range table at offset 0x0 has a premature terminator entry at offset 0x10
[ 16%] Linking C static library libdp-fconv2d.a
make[3]: Leaving directory '/spatz/hw/system/spatz_cluster/sw/build'
[ 16%] Built target snRuntime-cluster
```
not sure if this is related

**Comment by suehtamacv** [comment]
There was an issue with the runtime. I solved it with #6, thanks for reporting the issue!
